requirements.txt

```diff
@@ -0,0 +1,30 @@
+autopep8==1.6.0
+black==21.12b0
+click==8.0.3
+cycler==0.11.0
+dataclasses==0.8
+importlib-metadata==4.8.3
+joblib==1.1.0
+kiwisolver==1.3.1
```

Comment:
same: make sure we limit to the exact set of libs required

Reply:
Updated requirements.txt in commit [8292dec](https://github.com/criteo-research/criteo-privacy-preserving-ml-competition/pull/1/commits/8292decb0bd1180f331e29900c6644470bcfd3d2).